### PR TITLE
Logging In

### PR DESCRIPTION
**Pull request by SamYusuf**
Resolves loggin in
https://trello.com/c/BmkqMm4Q

### CODE REVIEW

**Comment by tbd-develop** [comment]
[Logging In](https://trello.com/c/BmkqMm4Q/5-logging-in)

File: src/app/Controllers/AuthController.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using app.Data.Models;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+
+//TODO: move to data models folder
```
Comment (playdeezgames) [inline src/app/Controllers/AuthController.cs]: Why TODO? just do!

File: src/app/appsettings.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=PetHome;Trusted_Connection=True;MultipleActiveResultSets=true"
```
Comment (playdeezgames) [inline src/app/appsettings.json]: BAD BAD BAD BAD BAD BAD
(no twinkie)

File: src/app/Controllers/AuthController.cs (hunk 1 above)
Comment (tbd-develop) [reply src/app/Controllers/AuthController.cs]: Data models library isn't defined yet and hasn't been discussed on stream as to how we'll lay out the application

File: src/app/Data/Migrations/20190708000727_Inital.cs [hunk 3]
```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace app.Data.Migrations
```
Comment (playdeezgames) [inline src/app/Data/Migrations/20190708000727_Inital.cs]: idiom?
the usual idiom for C# is initial casing
is there sufficient reason to not do this?

File: src/app/appsettings.json (hunk 2 above)
Comment (playdeezgames) [reply src/app/appsettings.json]: user secrets is a better way
or a app.local.settings file that is not checked in

Comment (tbd-develop) [reply src/app/appsettings.json]: What is the correct way of doing this, how should we approach this, please provide sources of information?

File: src/app/Data/Migrations/PetHomeContextModelSnapshot.cs [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+using app.Data;
+
+namespace app.Data.Migrations
+{
+    [DbContext(typeof(PetHomeContext))]
+    partial class PetHomeContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
```
Comment (playdeezgames) [inline src/app/Data/Migrations/PetHomeContextModelSnapshot.cs]: i assume this is generated code? if it isn't, then #pragmas should have comments with justifications.

File: src/app/Data/Migrations/20190708000727_Inital.cs (hunk 3 above)
Comment (tbd-develop) [reply src/app/Data/Migrations/20190708000727_Inital.cs]: Not discussed this yet as a group, how we're going to namespace. We'll likely stick to style standards.

**Pull request closed without merging**